string[] body_names
float64[] noise
bool[] hold_6dof_children
---
#bool[] ok
